packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -67,6 +74,41 @@ const AddEntityHelpTip: FC<{ parentageLabel?: string }> = memo(({ parentageLabel
         </>
     );
 });
+
+export const UnitKinds: Record<UNITS_KIND, UnitKindType> = {
+    [UNITS_KIND.NONE]: {
+        value: UNITS_KIND.NONE,
+        label: 'Any',
+        hideSubSelect: true,
+        msg: 'Amounts can be entered in any unit and won’t be converted.',
```

Comment:
Perhaps "won't be converted when stored or displayed?" (I'm not sure users know when or why conversions happen.)